chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/pdf/SummaryStatisticsSectionPdf.java

```diff
@@ -81,8 +81,8 @@ public Table addTableRow(Table table, List<String> cells, Boolean bold) {
 
     public List<String> createDataForRow(String description, Long recentValue, Long previousValue) {
         return Arrays.asList(description,
-                recentValue.toString(),
-                previousValue.toString(),
+                recentValue != null ? recentValue.toString() : "Not Available",
+                previousValue != null ? previousValue.toString() : "Not Available",
                 getDelta(recentValue, previousValue));
```

Comment:
What happens if either recentValue or previousValue are null, in the "getDelta" method?